deployment/src/test/java/io/quarkiverse/quinoa/test/QuinoaPackageManagerInstallPrependBinaryTest.java

```diff
@@ -41,7 +41,7 @@ public void testQuinoa() {
     }
 
     private static String convertToWindowsPathIfNeeded(String path) {
-        return isWindows() ? PackageManagerInstall.convertToWindowsPathIfNeeded(path) : path;
+        return isWindows() ? PackageManagerInstall.normalizePath(path) : path;
```

Comment:
This is seems to brake the tests

Reply:
The entire command was being normalized, not just the path. I have wrapped just the path to the command in the utility method